x-pack/test/functional/apps/canvas/feature_controls/canvas_spaces.ts

```diff
@@ -14,18 +14,29 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
   const PageObjects = getPageObjects(['common', 'canvas', 'security', 'spaceSelector']);
   const appsMenu = getService('appsMenu');
   const testSubjects = getService('testSubjects');
+  const kibanaServer = getService('kibanaServer');
+  const soInfo = getService('savedObjectInfo');
+  const log = getService('log');
 
   describe('spaces feature controls', function () {
     this.tags(['skipFirefox']);
+
     before(async () => {
       await esArchiver.loadIfNeeded('x-pack/test/functional/es_archives/logstash_functional');
     });
 
+    after(async () => {
+      await kibanaServer.savedObjects.clean({ types: ['canvas-workpad'] });
+      await soInfo.logSoTypes(log);
```

Comment:
Is this just some debug logging or a part of the test?

Reply:
It's just debug logging.  I left it here for posterity (so we remember it's there, and most importantly, what it does.  That said, I've no issue dropping it! :). Thanks @poffdeluxe